imports/plugins/included/taxes-avalara/client/accounts/exemption.js

```diff
@@ -6,11 +6,16 @@ import { Packages, Accounts } from "/lib/collections";
 import { Accounts as AccountsSchema } from "/lib/collections/schemas/accounts";
 import { TaxEntityCodes } from "/client/collections";
 
+let entityCodeList = [];
+let currentAccount;
+
 Template.taxSettingsPanel.helpers({
   account() {
     const sub = Meteor.subscribe("UserAccount", this.member.userId);
```

Comment:
Yes. I believe you mentioned this before. 

The reason the subscription was done here is because when the subscription is done outside of the `account()` helper, Autoform doesn't pick the new member data change that happens when the "Manage" button is clicked for another member.

Hence the form data remains at the very first member info that was clicked when the taxsetting form was created.

But I'll confirm this again and update.